renovate.json

```diff
@@ -3,14 +3,15 @@
   "extends": [
     "config:base"
   ],
-  "reviewers": ["team:data-platform-postgresql"],
+  "reviewers": ["dragomirp", "marceloneppel", "taurus-forever"],
```

Comment:
I think this is more useful than just assigning the team.

Reply:
I wish to use team, but it doesn't work well. LGTM for now.